kolibri/core/device/serializers.py

```diff
@@ -16,11 +16,14 @@
 from kolibri.core.device.utils import provision_device
 from kolibri.core.device.utils import provision_single_user_device
 from kolibri.core.device.utils import valid_app_key_on_request
+from kolibri.core.tasks.main import job_storage
 from kolibri.plugins.app.utils import GET_OS_USER
 from kolibri.plugins.app.utils import interface
 from kolibri.utils.filesystem import check_is_directory
 from kolibri.utils.filesystem import get_path_permission
 
+SYNC_CANCEL_STATIC_ID = "783"
```

Comment:
I think the point of defining a constant is that there's one definition for the ID, but it appears you've defined this in more than one place! I would suggest having this import the ID from the `tasks` module